test/react/first-render-only/react-context3.js

```diff
@@ -2,7 +2,12 @@ var React = require('React');
 // the JSX transform converts to React, so we need to add it back in
 this['React'] = React;
 var { Provider, Consumer } = React.createContext(null);
-// this is done otherwise the test fails
+// this is done otherwise the test fails because getTrials
```

Comment:
You can remove these changes after https://github.com/facebook/prepack/pull/2102 lands